src/helix/index.html

```diff
@@ -82,7 +82,7 @@
             window.location.href = link;
           }
         }).catch(() => {
-          document.getElementById("content").innerHTML = "<h3 style='color: red;'>Unable to Load Helix Data</h3>"
+          document.getElementById("content").innerHTML = "<h3 style='color: red;'>This is not a helix test, please use the other tabs</h3>"
```

Comment:
Is there a specific tab to look at? I expect still some levels of confusion if we're telling people to just poke around in other tabs.

Reply:
Will this also hide real errors reaching Helix by the tab?

Reply:
Or maybe the "catch" also triggers from the "then"? I'm not terribly familiar with JS promise land.  @alexperovich would probably know.

Reply:
there are specific tabs as Debug, Attachment, History, I can mention those. 

as far as I know no errors should be hidden for this. 
I had observed that most errors manifest of the tab not getting populated at all and when is not able to find the work item it returns 404, like in the following example (this work item is process on a staging queue so is not able to find it)
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/6895254/169140878-5fa6db17-76a3-474e-9bd8-b3898aa70957.png">


but probably I am missing some scenarios @ChadNedzlek / @garath  do you know any scenario in which this change can hide real errors?

Reply:
> there are specific tabs as Debug, Attachment, History, I can mention those.

I guess my question is what is our guidance. Where should people look for the artifacts they were looking for in the helix tab for a non-helix test? 

A user goes to the artifacts tab for a specific reason, to find the test's artifacts. Mentioning a random assortment of other tabs doesn't seem like it would make for a pleasant experience.

If the is issue that we don't know where the artifacts for non-helix tests might be, I think the message should just be that the artifacts for this test are not available in this tab because it's not a helix test. I know I'd be frustrated if the tab tells me to dig around in the history tab, where I will definitely not find what I was looking for.

Reply:
I'm pretty sure the `catch` includes the `then`. 

Since the focus is improving messaging it would be great to distinguish between "the comment parses like a proper helix test but helix didn't respond or didn't respond like I expected" and "the comment does not parse like a helix test, therefore this is not a helix test." (But of course don't talk about the comment field itself.)

Reply:
I agree with @riarenas  (that it should probably just say "This isn't a helix test, so there are no artifacts", probably without mentioning the other tabs at all).. And @garath that "there is no helix comment" might want to be a specific thing, rather than just catching any error that comes out of this.  I think that would mean either "result.comment" is null, or it doesn't have the 2 fields we are looking for.